buildconfig/Jenkins/buildscript

```diff
@@ -342,7 +342,9 @@ fi
 userconfig_dir=$HOME/.mantid
 rm -fr $userconfig_dir
 mkdir -p $userconfig_dir
-touch $userconfig_dir/Mantid.user.properties
+# use a fixed number of openmp threads to avoid overloading the system
+userprops_file=$userconfig_dir/Mantid.user.properties
+echo MultiThreaded.MaxCores=2 > $userprops_file
```

Comment:
I thought the [recommendation](https://github.com/DMSC-Instrument-Data/documents/blob/eef67316a6c88d66cc1873ce2431c5646045a2d2/investigations/Ctest_and_threads/using_threads_with_ctest.md) was to set this to be `${BUILD_THREADS}/4`. The evaluation would be
```shell
echo "MultiThreaded.MaxCores=$((${BUILD_THREADS} / 4))"
```

Reply:
I was going with the solution for maximum performance: https://github.com/DMSC-Instrument-Data/documents/blob/eef67316a6c88d66cc1873ce2431c5646045a2d2/investigations/Ctest_and_threads/using_threads_with_ctest.md#maximum-performance